qa/tasks/qemu.py

```diff
@@ -285,8 +285,9 @@ def _setup_nfs_mount(remote, client, service_name, mount_dir):
     export = "{dir} *(rw,no_root_squash,no_subtree_check,insecure)".format(
         dir=export_dir
     )
+    delete_mount_point = '/' + client + '/d'
```

Comment:
Shouldn't this just be derived from `export_dir`? Just swap out the `sed` delimiters from `/` to something like `|`

Reply:
hi, jason. 
there is a race between different RBD client on the same teuthology node. 

For example, When there are two RBD clients on the same teuthology-node, two mount points should be inserted to /etc/exports. These two mount points should be 
/export/client.0 *(rw,no_root_squash,no_subtree_check,insecure)
/export/client.1 *(rw,no_root_squash,no_subtree_check,insecure)
 
But, when executing  setup_nfs_mount function, "sudo sed -i  /^\/export\//d /etc/exports" always delete the all  mount points. So, when inserting client.1, client.0 will be removed. The result is that there just exist one mount point.  In the other words,  client.0 mount point always  be removed by client.1 when inserting client.1 mount point. 

I have modified coed to use export_dir to determine which mount point should be removed.

Reply:
Thanks. I understood the problem but was asking for a cleaner change (i.e. just re-use `export_dir` in the `sed` call to avoid essentially re-creating it). Since `export_dir` includes embedded `/` characters, you just need to switch `sed` over to use a different delimiter from `/`.

Reply:
got your points. modified it. thanks jason